jwst/pipeline/calwebb_image2.py

```diff
@@ -10,6 +10,7 @@
 from ..assign_wcs import assign_wcs_step
 from ..flatfield import flat_field_step
 from ..photom import photom_step
+from ..resample import resample_step
 
 
 __version__ = "3.0"
```

Comment:
This version number is probably in conflict with what's in the repo now (0.8.0).